.circleci/config.yml

```diff
@@ -91,6 +126,8 @@ workflows:
     jobs:
       - run-specs-with-postgres
       - run-specs-with-mysql
+      - run-specs-with-postgres-and-solidus-master
+      - run-specs-with-mysql-and-solidus-master
```

Comment:
We now have two sets of jobs: the first two jobs are for the current Solidus version and the next two jobs are for Solidus master.

Reply:
Maybe we can rename the other two jobs into `run-specs-with-postgres-and-solidus-latest` to be more explicit?